server/vulnerabilities/cpe.go

```diff
@@ -254,7 +254,7 @@ func TranslateSoftwareToCPE(
 			continue
 		}
 		if cpe == "" {
-			continue
+			cpe = fmt.Sprintf("none:%d", software.ID)
```

Comment:
`if software.Vendor == "Ubuntu"`? (we would need to set the `software.Vendor` to `"Ubuntu"` when ingesting software in `directIngestSoftware`).

Reply:
Yeah, I gave it some thought after the weekly meeting call, and I couldn't think of any good reason for doing that? (might be wrong) - the only place where having a dummy CPE might be undesired is when matching from CPE -> CVE (and we are excluding CPEs that came from `ubuntu` ATM), and even if some dummy CPE ends un the CPE -> CVE process, nothing will happen (because we will get no CVE match).